### PR TITLE
Temporary fix for issue Glimesh#383

### PR DESCRIPTION
**Pull request by Drehd**
# Before:
image

# After:
image

### CODE REVIEW

**Comment by clone1018** [comment]
Couple of test errors here @Drehd. Should be simple text matches :D

**Comment by Drehd** [comment]
Noticed on even smaller displays the buttons still would be one on top of the other, so I made the button resize to prevent that in the last commit.

File: assets/css/glimesh/components/stream.scss [hunk 1]
```diff
@@ -24,4 +24,37 @@
...
     }
+}
+
+@media (max-width: 768px) {
+    .btn-responsive {
+        padding: 2px 4px;
+        line-height: 1;
+        border-radius: 3px;
+    }
+    .f_txt {
```
Comment (clone1018) [inline assets/css/glimesh/components/stream.scss:35]: What do you think about replacing these with a common class you can add to each element you want to hide? 

Optionally Bootstrap provides these classes we can use to achieve the same effect with the Bootstrap best practices included: https://getbootstrap.com/docs/4.6/utilities/display/#hiding-elements

**Comment by clone1018** [comment]
Thanks for the work, I tested this out locally and it works beautifully! Just have a comment on the CSS.

File: assets/css/glimesh/components/stream.scss (hunk 1 above)
Comment (Drehd) [reply assets/css/glimesh/components/stream.scss:35]: That looks way better, going to try that now, thank you!

Comment (clone1018) [reply assets/css/glimesh/components/stream.scss]: Looks awesome!

**Review by clone1018 — APPROVED** [review]

**Pull request merged**